### ISSUE TITLE
Font size goes back to default if sign out/in again.

### ISSUE THREAD

**Issue by HubKing** [vector-im/element-web#25220]
### Steps to reproduce

If I set the largest font size like so:

![image](https://user-images.githubusercontent.com/10900489/234843265-c8beba89-63ff-4a4d-9205-d89b9b68b029.png)

and sign out and sign in again, the font size goes back to the default size.

#### What did you expect?
Font size should be remembered.

#### What happened instead?
Font size goes back to default.

### Operating system

Arch Linux

### Application version

Element version: 1.11.26 Olm version: 3.2.12

### How did you install the app?

Arch repository

### Homeserver

matrix.org

### Will you send logs?

No

**Comment by t3chguy** [comment]
Yup, all settings and data is cleared for privacy when you sign out.

**Comment by HubKing** [comment]
> Yup, all settings and data is cleared for privacy when you sign out.

Seems kind of extreme. I only signed out to try another server, not to wipe out my settings. Also, things like font settings aren't very confidential. Shouldn't there be some sort of way/option not to clear settings when logging out?

**Comment by t3chguy** [comment]
Font Size can be stored in your account settings (Element defaults to writing it to device settings but will read from both) so if settings weren't cleared on log-out this would not work as expected.
